### PR TITLE
Create initial CHM help documentation, tooling, and System Explorer integration

### PR DESCRIPTION
**Pull request by Karl-G1**
### What does this Pull Request accomplish?

This implements the same CHM infrastructure for this custom device as is used in the [Routing and Faulting CD](https://github.com/ni/niveristand-routing-and-faulting-custom-device). This particular submission does the following:

- Create CHM tooling to generate built CHM file
- Create HTML files for main page and monitoring page
- Adds a VI to auto-generate CHM on build
- Modifies the main page and monitoring page to use the CHM pages
- Adds a helper VI to load the correct CHM page

### Why should this Pull Request be merged?

Having some documentation on the main page is a huge lift to the custom device usability. It will also give @dbendele an easy way to add further documentation to custom device pages.

image

### What testing has been done?

Manually built the Configuration Release build specification and tested the resulting files in VS 2017.

### CODE REVIEW

**Comment by rtzoeller** [comment]
image
Any reason for the blue bar when the CHM shows the custom device name?

File: VeriStand/Data Sharing Custom Device/Source/Help/Build CHM.log [hunk 1]
```diff
@@ -0,0 +1,28 @@
+Microsoft HTML Help Compiler 4.74.8702
```
Comment (rtzoeller) [inline VeriStand/Data Sharing Custom Device/Source/Help/Build CHM.log:1]: I feel like this file should be listed in a `.gitignore` file.

Git supports .gitignore files at any level in the folder structure, so you can put one in `Data Sharing Custom Device/Source/Help` and avoid cluttering the top level one, if you prefer.

Comment (Karl-G1) [reply VeriStand/Data Sharing Custom Device/Source/Help/Build CHM.log]: Done. Deleted file and added a .gitignore for them.

**Comment by Karl-G1** [comment]
> image
> Any reason for the blue bar when the CHM shows the custom device name?

No reason. Removed the blue bar.

**Comment by dbendele** [comment]
DSF System Explorer.lvlib--Load Help File.vi
- [ ] change defaults on front panel to point to DSF chm and html.

**Comment by dbendele** [comment]
LLB Pre-Build CHM Build.vi
- [ ] this VI has no **error in** input. Should it have an **error in** input to skip building of chm on case where there is an input error?

**Comment by rtzoeller** [comment]
> [ ] this VI has no **error in** input. Should it have an **error in** input to skip building of chm on case where there is an input error?

I don't think it can. The template for pre-build actions does not include one, and includes the following note:
image

**Pull request merged**